pkg/services/alerting/notifiers/hipchat.go

```diff
@@ -37,6 +37,15 @@ func init() {
           data-placement="right">
         </input>
       </div>
+			<div class="gf-form max-width-30">
+        <gf-form-switch
+           class="gf-form max-width-30"
+           label="Plain Text"
+           label-class="width-8"
+           checked="ctrl.model.settings.plainText"
+           tooltip="Send Hip Chat messages as text instead of the default card (Allows for more detailed messages).">
```

Comment:
But if the card override the plain text cannot Grafana send both? So no option is needed?